src/main/java/bio/terra/janitor/db/TrackedResourceId.java

```diff
@@ -0,0 +1,20 @@
+package bio.terra.janitor.db;
+
+import java.util.UUID;
+
+/** Wraps the tracked_resource_id in db tracked_resource table. */
+public class TrackedResourceId {
```

Comment:
adding a equals and hash method would be nice. Could just make this an autovalue.